src/gordon_gcp/clients/http.py

```diff
@@ -19,10 +19,6 @@
 product-specific API clients as it handles Google authentication and
 automatic refresh of tokens.
 
-.. todo::
-
-    Include that it also handles retries once implemented.
```

Comment:
Could you update the module-level docs to reflect the new functionality?

Reply:
Will fix.